oamapps/postConfigure/postConfigure.cpp

```diff
@@ -3880,17 +3905,7 @@ int main(int argc, char* argv[])
     //check if local MariaDB ColumnStore system logging is working
     cout << endl << "===== Checking MariaDB ColumnStore System Logging Functionality =====" << endl << endl;
 
-    if ( rootUser)
-        cmd = installDir + "/bin/syslogSetup.sh install  > /dev/null 2>&1";
-    else
-        cmd = "sudo " + installDir + "/bin/syslogSetup.sh --installdir=" + installDir + " install  > /dev/null 2>&1";
-
-    system(cmd.c_str());
-
-    if ( rootUser)
-        cmd = installDir + "/bin/syslogSetup.sh status  > /dev/null 2>&1";
-    else
-        cmd = "sudo " + installDir + "/bin/syslogSetup.sh --installdir=" + installDir + " status  > /dev/null 2>&1";
+    cmd = installDir + "/bin/syslogSetup.sh status  > /dev/null 2>&1";
 
     int ret = system(cmd.c_str());
```

Comment:
Doesn't the installdir need to be specified to syslogSetup.sh as it is now?  I'd have to re-read that script to see if the 'status' cmd really needs root perms.  But in general, if the sudo was actually necessary before, then it won't work as not-root if you remove sudo.